packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -184,11 +193,15 @@ class _DevToolsMenu extends StatelessWidget {
   }
 
   bool _shouldIncludeScreen(ScreenMetaData screen) {
-    // Don't show home screen or debugger in the menu.
+    // Some screens shouldn't show up in the menu.
     return screen != ScreenMetaData.home &&
         screen != ScreenMetaData.debugger &&
+        screen != ScreenMetaData.vmTools &&
```

Comment:
can you add a TODO here to consider showing this conditionally based on `preferences.vmDeveloperModeEnabled`? Or we could do that here if it is simple